packages/pigeon/lib/dart_generator.dart

```diff
@@ -369,8 +380,9 @@ void _writeFlutterApi(
                         '$leftHandSide = ($argsArray[$count] as $genericArgType?)${castCall.isEmpty ? '' : '?$castCall'};');
                   }
                   if (!arg.type.isNullable) {
+                    indent.writeln('assert($argName != null,');
                     indent.writeln(
-                        "assert($argName != null, 'Argument for $channelName was null, expected non-null $argType.');");
+                        "'Argument for $channelName was null, expected non-null $argType.');");
```

Comment:
Optional nit: Does breaking this up actually help? I would think the indentation on the second line would be wrong.